worker/src/RTC/TransportCongestionControlServer.cpp

```diff
@@ -153,13 +208,11 @@ namespace RTC
 
 					case RTC::RTCP::FeedbackRtpTransportPacket::AddPacketResult::MAX_SIZE_EXCEEDED:
 					{
-						// Send ongoing feedback packet and add the new packet info to the
-						// regenerated one.
-						SendTransportCcFeedback();
-
-						this->transportCcFeedbackPacket->AddPacket(wideSeqNumber, nowMs, this->maxRtcpPacketLen);
-
-						break;
+						// This should not happen
+						MS_DEBUG_DEV("transport-cc feedback packet is exceeded!");
+						// Create a new feedback packet.
+						this->transportCcFeedbackPacket.reset(new RTC::RTCP::FeedbackRtpTransportPacket(
```

Comment:
Always blank line after logs. And let's use warn here since it looks more like a warning. And add missing dot.

```suggestion
						// This should not happen.
						MS_WARN_DEV("transport-cc feedback packet is exceeded");

						// Create a new feedback packet.
						this->transportCcFeedbackPacket.reset(new RTC::RTCP::FeedbackRtpTransportPacket(
```